esx_service/vmdk_ops.py

```diff
@@ -598,13 +598,15 @@ def removeVMDK(vmdk_path, vol_name=None, vm_name=None, tenant_uuid=None, datasto
     logging.info("*** removeVMDK: %s", vmdk_path)
 
     # Check the current volume status
-    kv_status_attached, kv_uuid, attach_mode = getStatusAttached(vmdk_path)
+    kv_status_attached, kv_uuid, attach_mode, attached_vm_name = getStatusAttached(vmdk_path)
     if kv_status_attached:
         ret = handle_stale_attach(vmdk_path, kv_uuid)
         if ret:
-            logging.info("*** removeVMDK: %s is in use, VM uuid = %s (%s)", vmdk_path, kv_uuid, ret)
-            return err("Failed to remove volume {0}, in use by VM uuid = {1}.".format(
-                vmdk_path, kv_uuid))
+            if vol_name is None:
```

Comment:
I had to add this because the function signature is `removeVMDK(.., vol_name=None, ..)` and there are a few places from where it's called without the `vol_name` parameter. Making the suggested change will need a modification at [vmdk_ops.py#L355](https://github.com/venilnoronha/docker-volume-vsphere/blob/1a8bbb75cb2034efed0c4230066cffbde7d8a694/esx_service/vmdk_ops.py#L355) and at multiple locations in [vmdk_ops_test.py](https://github.com/venilnoronha/docker-volume-vsphere/blob/1a8bbb75cb2034efed0c4230066cffbde7d8a694/esx_service/vmdk_ops_test.py). I can address that.